views/index.jade

```diff
@@ -13,21 +13,21 @@ block content
 
             #bs-example-navbar-collapse-1.collapse.navbar-collapse
                 ul.nav.navbar-nav
-                    - if(login_server && user.username !== 'admin')
+                    - if(login && user.username !== 'admin')
                         li
                             a(href="/user/library") Library
                         li
                             a(href="/user/playlist") Playlist
 
-                    - else if(login_server && user.username === 'admin')
+                    - else if(login && user.username === 'admin')
```

Comment:
eventually we want to not hardcode the username check and switch to a role check.